cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -68,8 +68,8 @@
  *
  * @since 0.90
  */
-@CommandLine.Command(name = BUILD_COMMAND, description = "Ballerina build - Builds Ballerina module(s) and generates " +
-                                                         "executable outputs.")
+@CommandLine.Command(name = BUILD_COMMAND, description = "Ballerina build - Build Ballerina module(s) and generate " +
+                                                         "executable output.")
```

Comment:
This is sorta wrong, even the previous message. The binary built by the `build` command is not necessarily executable in the sense that it doesn't have to have an entry point.

Reply:
As per the offline discussion we had with Kishanthan, shall we mention in the help text of build that executables are produced only for modules containing an entry point?

Reply:
Fixed in the cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help file.